examples/file-server/src/file-actions.ts

```diff
@@ -15,7 +15,7 @@ export class FileActions {
 
     /**
      * will return the relative path of the file and the name of the file
-     * @param filePath path to the file in the local file system
+     * @param filePath - path to the file in the local file system
```

Comment:
the prefix dash causes it to show as a list item in auto completions. this is a regression.

Reply:
as we discussed, theta's the TSDocs spec. I can parse it and take the warnings if that's what's we decide